src/features/projects/styles/ProjectDetails.module.scss

```diff
@@ -264,6 +264,10 @@
   .container {
     padding-bottom: 0px;
   }
+
+  .projectContainer {
+    padding-bottom: 80px;
+  }
 }
 
 @include mdTabletView {
```

Comment:
Resolve codefactor error by replacing margin: 10px by margin-left: 10px or swap lines 282 ( margin-top: 0px;) and 284 (margin: 10px;)